test/test_images/runtime/handlers/user.go

```diff
@@ -23,9 +23,16 @@ import (
 )
 
 func userInfo() *types.UserInfo {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return &types.UserInfo{Error: err.Error()}
```

Comment:
Having the error retrieving cwd affect all of UserInfo means that the both this and the user_test will fail together making this harder to debug. If would be better to have the error condition associated with just cwd.

Reply:
Makes sense, I misread some of the other types. Adjusted.